src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -1076,3 +1076,23 @@ PARAM_DEFINE_INT32(FW_LND_NUDGE, 2);
  * @group FW Auto Landing
  */
 PARAM_DEFINE_INT32(FW_LND_ABORT, 3);
+
+/**
+ * Wind-based airspeed scaling factor
+ *
+ * Multiplying this factor with the current absolute wind estimate gives the airspeed offset
+ * added to the minimum airspeed setpoint limit. This helps to make the
+ * system more robust against disturbances (turbulence) in high wind.
+ * The airspeed increase is limited to 3 m/s.
```

Comment:
Alternatively you could directly parameterize the max (3 m/s)?

Reply:
Don't really follow, can you elaborate?